src/liblelantus/sigmaextended_verifier.cpp

```diff
@@ -88,10 +84,9 @@ bool  SigmaExtendedVerifier::verify(
         Scalar fi_sum(uint64_t(0));
         for (uint64_t i = I[j] + 1; i < n; i++)
             fi_sum += f_[j*n + i];
-        pow += fi_sum * xj.pow * f_part_product[m - j - 1];
+        f_i_[N - 1] += fi_sum * xj.pow * f_part_product[m - j - 1];
```

Comment:
is `pow` and `f_i_[N-1]` the same before the loop? They are calculated in two independent ways. I believe it's clearer (and slightly faster) to leave the code here as is and then assign instead of emplacing.

Reply:
Yes `pow` and `f_i_[N-1]` are the same before this step. I don't think it will change something in terms of performance, but yea may be it is more clear. will change it